drivers/scheduler/k8s/specs/cassandra-backup/pxd/px-storage-class.yaml

```diff
@@ -0,0 +1,12 @@
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: portworx-sc
+provisioner: kubernetes.io/portworx-volume
+parameters:
+  {{ if .Repl }}
+  repl: "{{ .Repl }}"
+  {{ else }}
+  repl: "3"{{ end }}
+  priority_io: "high"
+  snap_schedule: "periodic=60,5"
```

Comment:
Please add a blank line at the end of file

Reply:
Done